dags/utils/acoustic/acoustic_client.py

```diff
@@ -187,3 +184,53 @@ def generate_report(
         logging.info(f"{report_type} generation complete. Report location: {report_loc}. Time taken: {datetime.now() - start}")
 
         return
+
+
+if __name__ == "__main__":
+
+    EXEC_START = "01/04/2022 00:00:00"
+    EXEC_END = "02/04/2022 00:00:00"
```

Comment:
This is for testing correct? Since this client is primarily used as a module? Might be worth mentioning that here (or putting this in a test?)

Reply:
yeah, to be honest this was just for me generating a report locally so that I can check it worked without having to go through a PR process.

Reply:
will remove later.